db/warehouse/migrate/20250304143108_add_project_id_to_hud_clients.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddProjectIdToHudClients < ActiveRecord::Migration[7.0]
+  def change
+    ['hud_report_dq_clients', 'hud_report_path_clients'].each do |table|
+      add_column table, :project_id, :integer
```

Comment:
My understanding is that Todd is migrating us away from integer pks on production as the part of pg/rds updates.
```suggestion
      add_reference table, :project
```